en/identity-server/6.1.0/docs/assets/code-samples/localize.jsp

```diff
@@ -0,0 +1,181 @@
+<%--
```

Comment:
Do we really need the license?

Reply:
And is it necessary to have this sample, can't we just ask the users to copy the localize.jsp file as we did for other files.